ivy/functional/ivy/data_type.py

```diff
@@ -817,7 +817,7 @@ def result_type(
     >>> y = ivy.array([3., 4., 5.])
     >>> d = ivy.result_type(x, y)
     >>> print(d)
-    float32
+    float64
```

Comment:
`float32` is the default dtype so ideally it would create the arrays with `float32` dtype in the result as well, probably there's some problems with the code. We can ofc mark this failure in the skip list if it's a more fundamental issue than just the docstrings.